modules/core/src/main/java/org/apache/ignite/internal/processors/cache/distributed/dht/topology/GridDhtPartitionsStateValidator.java

```diff
@@ -328,4 +338,59 @@ private String fold(AffinityTopologyVersion topVer, Map<Integer, Map<UUID, Long>
 
         return sb.toString();
     }
+
+    /**
+     * Folds given maps of invalid partition states to string (Both PartitionCounters and Size) representation
+     * in the following format:
+     * Part [id]: [consistentId=value meta=[updCnt=value, size=value]]
+     * input parameters are maps of invalid partition update counters and sizes,
+     * return value is String in the following format:
+     * Part [id]: [consistentId=value meta=[updCnt=value, size=value]]
+     */
```

Comment:
Please correct the javadoc, with the addition of @param and @return, as for example here

org.apache.ignite.internal.processors.cache.distributed.dht.topology.GridDhtPartitionsStateValidator#fold(org.apache.ignite.internal.processors.affinity.AffinityTopologyVersion, java.util.Map<java.lang.Integer,java.util.Map<java.util.UUID,java.lang.Long>>)